src/hash_types.rs

```diff
@@ -55,9 +52,3 @@ impl_hashencode!(Wtxid);
 impl_hashencode!(SigHash);
 impl_hashencode!(BlockHash);
 impl_hashencode!(TxMerkleNode);
-
-impl ThirtyTwoByteHash for SigHash {
```

Comment:
If this is deleted here, we should just drop the previous commit.

Reply:
I must have messed something up, because this change was desired. I'll fix it.